ci/jira_query.py

```diff
@@ -27,8 +27,8 @@
 # Expected components from the commit message, and directory in src/, src/client or utils/ is also
 # valid.  We've never checked/enforced these before so there have been a lot of values used in the
 # past.
-VALID_COMPONENTS = ('agent', 'build', 'ci', 'csum', 'doc', 'gha', 'il', 'md', 'mercury', 'swim',
-                    'test', 'tools')
+VALID_COMPONENTS = ('agent', 'build', 'ci', 'csum', 'doc', 'gha', 'il', 'md', 'mercury', 'pil4dfs',
+                    'swim', 'test', 'tools')
```

Comment:
There's nothing formal about this, right? It's somewhat arbitrary what we allow?

Reply:
Traditionally there was nothing formal about it and all kinds of values was used, this is the only place it's checked and we do seem to be sticking to the known components now.  Note that it'll also accept the names of directories in src or src/client, this list isn't all the acceptable values.

We don't yet use this value for anything else like selecting testing which I think we could probably start doing now, and we don't make any attempt to verify that the component is broadly correct.